README.md

```diff
@@ -1 +1,116 @@
 # python-esiclient
+
+`python-esiclient` extends the OpenStack CLI to provide simplified user workflows, encapsulating multiple OpenStack CLI commands into a single CLI command.
+
+## `openstack esi node network <command>`
+
+These commands manage network connections to nodes.
+
+### `openstack esi node network list`
+
+List node/network attachments.
+
+```
+   openstack esi node network list
+       [--node <node>]
+       [--network <network>]
+```
+
+**--node <node>**: Filter by node (name or UUID)
+**--network <network>**:  Filter by network (name or UUID)
+
```

Comment:
Do you want to include example output in this document?

Reply:
I would, but I actually don't have a good environment to produce this output at the moment.